Doc/whatsnew/3.9.rst

```diff
@@ -1165,3 +1171,7 @@ Removed
 
 * Remove ``_PyUnicode_ClearStaticStrings()`` function.
   (Contributed by Victor Stinner in :issue:`39465`.)
+
+* Remove ``Py_UNICODE_MATCH``.  It has been deprecated by :pep:`393`, and
+  broken since Python 3.3.
```

Comment:
Please add:

```
The :c:func:`PyUnicode_Tailmatch` function can be used instead.
```

It's not exactly the same, but it's the closest function that we have.

Reply:
Uh, I thought tailmatch is only prefix/suffix match.  Thank you.